examples/supercell.jl

```diff
@@ -0,0 +1,151 @@
+using Breeze
+using Oceananigans.Units
+using Statistics
+using Printf
+using CairoMakie
+
+using Oceananigans.Grids: znode
+using Oceananigans: Center, Face
+using Oceananigans.Operators: Δzᶜᶜᶜ, Δzᶜᶜᶠ, ℑzᵃᵃᶠ
+using Breeze.Thermodynamics: dry_air_gas_constant
+using CUDA
+
+using CloudMicrophysics
+import Breeze: Breeze
+
+# Access extension module and define aliases to avoid namespace conflicts
+const BreezeCloudMicrophysicsExt = Base.get_extension(Breeze, :BreezeCloudMicrophysicsExt)
+const BreezeOneMomentCloudMicrophysics  = BreezeCloudMicrophysicsExt.OneMomentCloudMicrophysics
+
+# Supercell simulation
+# Reference:
+# Klemp et al. (2015): "Idealized global nonhydrostatic atmospheric test cases on a reduced-radius sphere"
+
+#  grid configuration
+Nx, Ny, Nz = 336, 336, 40
+Lx, Ly, Lz = 168kilometers, 168kilometers, 20kilometers
+
+grid = RectilinearGrid(GPU(),
+                       size = (Nx, Ny, Nz),
+                       x = (0, Lx),
+                       y = (0, Ly),
+                       z = (0, Lz),
+                       halo = (5, 5, 5),
+                       topology = (Periodic, Periodic, Bounded))
+
+
+# Problem parameters and initial conditions
+pᵣ, θᵣ = 100000, 300
+thermo = ThermodynamicConstants()
+reference_state = ReferenceState(grid, thermo, base_pressure=pᵣ, potential_temperature=θᵣ)
+formulation = AnelasticFormulation(reference_state)
```

Comment:
should we use `thermodynamics=:LiquidIcePotentialTemperature` ?

probably we should make this default too

Reply:
Sounds good to me!